modules/_labs/color-picker/react/lib/parts/ColorReset.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import {borderRadius, colors, spacing, type, inputColors} from '@workday/canvas-kit-react-core';
+
+import {Swatch} from './Swatch';
+
+export const ResetButton: React.FunctionComponent<Props> = ({id, onClick, resetColor, label}) => {
```

Comment:
We found several issues using `React.FunctionComponent<Props>` or `React.FC<Props>` or `React.StatelessComponent<Props>` or any of the other signatures where Typescript doesn't understand `defaultProps` properly and our Storybook Docs can't extract Prop type tables properly...

```suggestion
export const ResetButton = ({id, onClick, resetColor, label}: Props) => {
```